grails-app/controllers/au/org/ala/ecodata/DocumentHostInterceptor.groovy

```diff
@@ -0,0 +1,36 @@
+package au.org.ala.ecodata
+
+import org.grails.web.servlet.mvc.GrailsWebRequest
+import org.springframework.web.context.request.RequestAttributes
+
+class DocumentHostInterceptor {
+    static final String DOCUMENT_HOST_NAME = "DOCUMENT_HOST_NAME"
+
+    HubService hubService
+    DocumentHostInterceptor () {
+        matchAll()
+    }
+
+    boolean before() {
+        String hostName = request.getHeader(grailsApplication.config.app.http.header.hostName)
+        if (hostName) {
```

Comment:
Do you think it's worth adding some validation to ensure the domain is a *.ala.org.au domain?  I can't think of a way this could be used to craft an attack but maybe safer to validate just in case?

Reply:
Made those changes